cloud_testing/kotlin_poc/src/test/kotlin/ftl/fixtures/flank.yml

```diff
@@ -1,5 +1,5 @@
 appApk: ../../test_app/apks/app-debug.apk
-testApk: ../../test_app/apks/app-debug-androidTest.apk
+testApk: gs://tmp_bucket_2/app-debug-androidTest.apk
```

Comment:
I think instead of replacing the local test apk, we should add a new fixture. This way both code paths are tested.

flank.local.yml
flank.gcs.yml

I think something like that would work ^